test/Rakefile

```diff
@@ -0,0 +1,78 @@
+# coding: utf-8
+require "jekyll"
+require "html-proofer"
+require "open3"
+
+def verify(site_dir)
+  options = {
+    :assume_extension => true,
+    :check_html => true,
+    :enforce_https => true,
+    :only_4xx => true,
+    :check_unrendered_link => true,
+  }
+  HTMLProofer.check_directory(site_dir, options).run
+end
+
+# Test generated output has valid HTML and links.
+task :test do
+  image_name = ENV.fetch("IMAGE_NAME", "swedbankpay/jekyll-plantuml")
+  image_tag = ENV.fetch("IMAGE_TAG", "latest")
+
+  # Spin up docker
+  # TODO: It should be enough to do `jekyll build` since HTMLProofer doesn't
```

Comment:
Is it a good to resolve this `TODO` now?

Reply:
Yes, indeed!